<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="Vive_Controller">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <pose>0 0 -0.1 0.4 0 1.57075</pose>
        <geometry>
          <mesh>
            <scale>0.0010 0.0010 0.0010</scale>
            <uri>model://Vive_Controller/meshes/HTC_Vive_controller.stl</uri>
          </mesh>
        </geometry>
      </collision>
      <visual name="visual">
        <pose>0 0 -0.1 0.4 0 1.57075</pose>
        <geometry>
          <mesh>
            <scale>0.0010 0.0010 0.0010</scale>
            <uri>model://Vive_Controller/meshes/HTC_Vive_controller.stl</uri>
          </mesh>
        </geometry>
         <material>
          <script>
            <uri>model://Vive_Controller/materials/gazebo.material</uri>
            <name>Gazebo/Grey</name>
          </script>
        </material>
      </visual>
    </link>
  </model>
</sdf>
